hrjobcontract/js/test/controllers/form/form-general_test.js

```diff
@@ -73,6 +73,15 @@ define([
         it('sets duration calculating number of days including start and end dates', function() {
           expect($scope.duration).toEqual('17 days');
         });
+
+        it('calculates duration from 2017-02-01 to 2017-03-31 as two motnhs', function() {
```

Comment:
I think this is too specific to be a good test name. It makes me think that it should only work for these 2 specific dates and not for everything else. Can you please make this more generic and try to actually describe what is the rule? As far as I understand, every 30 days should be considered a month, right?